src/Iso8601DurationHelper/Duration.cs

```diff
@@ -678,4 +678,32 @@ private enum DurationComponent
             Seconds = Minutes + 1
         }
     }
+
+    /// <summary>
+    /// Extension class to add functionality to <see cref="DateTime"/> and <see cref="DateTimeOffset"/> so that use of <see cref="Duration"/> is similar to using <see cref="TimeSpan"/>
+    /// </summary>
+    public static class DateTimeHelper 
```

Comment:
Could you please rename it to `DateTimeDurationExtensions`?

Reply:
Done